manifests/0000_90_openshift-apiserver-operator_05_check-endpoints_servicemonitor.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: openshift-apiserver-operator-check-endpoints
+  namespace: openshift-apiserver
+spec:
+  endpoints:
+  - bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+    interval: 30s
+    port: check-endpoints
+    scheme: https
+    tlsConfig:
+      insecureSkipVerify: true
```

Comment:
this one should be able to use the service-ca.

Also, we should have an e2e test that prevents these from being insecure.